setup.cfg

```diff
@@ -30,4 +30,29 @@ exclude =
   build/*,
   third_party/*
   *_pb2.py,
-  .cache/*
\ No newline at end of file
+  .cache/*
+
+[mypy]
+# follow-imports = silent  # TODO remove this
+mypy_path = stubs:third_party/pybind11
```

Comment:
What does this line do?

Reply:
It allows mypy to find typing information (`.pyi` files or `.py` files) in the `stubs/` directory and in the `third_party/pybind11` directory (`pybind11` for the corresponding module and `stubs` for some general local overrides)